### PR TITLE
InstructorFeedbackResultsPage shows students who did not give response only for the last team #2800

### LINKED ISSUE #2800: InstructorFeedbackResultsPage shows students who did not give response only for the last team
After #2741, on InstructorFeedbackResultsPage, show by GQR/GRQ, students who did not give responses should be shown. 

However, sometimes on teams that has a giver, the team members of that giver are sometimes not shown if they did not give a response. Also, the profile picture only appears when hovering over the giver's name, when it should appear when hovering over the email as well.

### PR DESCRIPTION
**Pull request by kanghj**
Fixes #2800

### CODE REVIEW

**Comment by kanghj** [comment]
@LowWeiLin

File: src/main/webapp/jsp/instructorFeedbackResultsByGiverQuestionRecipient.jsp [hunk 1]
```diff
@@ -214,6 +214,63 @@
...
+                        
+                        List<String> teamMembersList = new ArrayList<String>(teamMembersWithoutReceivingResponses);
+                        Collections.sort(teamMembersList);
+                        for (String email : teamMembersList) {
+        %>
+                            <div class="panel panel-primary">
+                            <div class="panel-heading">
+                                From: 
+                                <%
+                                if (validator.getInvalidityInfo(FieldValidator.FieldType.EMAIL, email).isEmpty()) {
```
Comment (LowWeiLin) [inline src/main/webapp/jsp/instructorFeedbackResultsByGiverQuestionRecipient.jsp]: Indentation

**Comment by kanghj** [comment]
Changes made @LowWeiLin

**Comment by LowWeiLin** [comment]
Looks okay

**Pull request merged**